src/main/kotlin/com/github/patou/gitmoji/GitCommitAction.kt

```diff
@@ -229,7 +229,7 @@ class GitCommitAction : AnAction() {
     private fun loadGitmojiFromHTTP() {
         val client = OkHttpClient().newBuilder().addInterceptor(SafeGuardInterceptor()).build()
         val request: Request = Builder()
-            .url("https://raw.githubusercontent.com/carloscuesta/gitmoji/master/src/data/gitmojis.json")
+            .url("https://raw.githubusercontent.com/carloscuesta/gitmoji/master/packages/gitmojis/src/gitmojis.json")
```

Comment:
There are now an api like describe in this readme :
https://github.com/carloscuesta/gitmoji/tree/master/packages/gitmojis

I think it's more good to use this url : https://gitmoji.dev/api/gitmojis


https://gitmoji.dev/api/gitmojis

Reply:
Agree